radix-engine-tests/tests/core.rs

```diff
@@ -34,3 +37,51 @@ fn test_call() {
     );
     receipt.expect_commit_success();
 }
+
+#[test]
```

Comment:
Can we add tests specifically for trying to globalize:
* Bucket/Vault/Proof
* Metadata/Royalty/RoleAssignment

Reply:
This would be more of product "behavior" tests rather than system tests

Reply:
Yes, I will add such tests in https://github.com/radixdlt/radixdlt-scrypto/pull/1268, because I added `is_transient` flag there for all the listed blueprints.